### ISSUE TITLE
Banner Image Pixelated

### ISSUE THREAD

**Issue by Sean12697** [Emanuell-Dan/manchester-vuejs#4]
If it's possible to be an SVG in a purple div, it should remove the pixelation and still have the same effect.

**Comment by Emanuell-Dan** [comment]
> If it's possible to be an SVG in a purple div, it should remove the pixelation and still have the same effect.

Yes, we will look to either do what you have suggested or maybe use a different banner altogether. Will follow-up with updates 🙂

**Comment by Emanuell-Dan** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Banner image quality](https://trello.com/c/AmRXdqIw/10-banner-image-quality)
